setup.py

```diff
@@ -57,14 +57,19 @@ def setup_path(path = ""):
 #        shutil.move(setup_path("website"), setup_path("docs"))
 
 if sys.platform == 'win32':
-    install_requires = ['six', 'pypiwin32', 'comtypes']
+    install_requires = ['six', 'comtypes']
+    try:
+        import win32api # check if it was already installed manually
+    except ImportError:
+        install_requires.append('pypiwin32')
+

```

Comment:
If you install pyWin32 manually from SourceForge, it's called `pywin32`, but on PyPI it's called `pypiwin32`. That's what makes pip (and people) crazy. :)